src/library/autodj/dlgautodj.cpp

```diff
@@ -150,7 +151,13 @@ DlgAutoDJ::DlgAutoDJ(WLibrary* parent,
             "\n"
             "Skip Silence Start Full Volume:\n"
             "The same as Skip Silence, but starting transitions with a centered\n"
-            "crossfader, so that the intro starts at full volume.\n");
+            "crossfader, so that the intro starts at full volume.\n"
+            "\n"
+            "Radio Laneway Crossfade:\n"
+            "Starts the next track at full volume.  Starts the crossfade when the\n"
+            "volume last falls below -12Db or at the spin box setting which ever\n"
+            "is lower, and potentially starts the next earlier if it starts below\n"
+            "-27Db.");
```

Comment:
its very easy to change the threshold in the analyzer but forget to update it here. Do you think its valuable to have the thresholds documented here or can we just omit them?

Reply:
I nearly made it a more general statement (e.g., 'a certain level'), but I figured someone would want to know the exact values. It may well be worth making it more general for the reason you suggested.